### ISSUE TITLE
problem about  outputs_coord

### ISSUE THREAD

**Issue by luotianhang** [roboflow/rf-detr#654]
```py
outputs_coord_delta = self.bbox_embed(hs)
outputs_coord_cxcy = outputs_coord_delta[..., :2] * ref_unsigmoid[..., 2:] + ref_unsigmoid[..., :2]
outputs_coord_wh = outputs_coord_delta[..., 2:].exp() * ref_unsigmoid[..., 2:]
outputs_coord = torch.concat(
  [outputs_coord_cxcy, outputs_coord_wh], dim=-1
)
```

I am working on this project ,and find a question ,the ref_unsigmoid is very unstable, and how to make sure all the value of outputs_coord is in [0,1],without a sigmoid

**Comment by isaacrob** [comment]
This is a fairly common reparameterization of bounding box regression in modern DETRs. It doesn't guarantee the coordinates are in 0-1, but has cleaner gradients. You can always clamp them in post processing.

**Comment by Borda** [comment]
@luotianhang — great question about the coordinate reparameterization! @isaacrob explained the design choice clearly: operating in unconstrained space gives cleaner gradients during training, even though values can technically exceed [0,1]. The tradeoff is intentional — gradient quality during training outweighs the convenience of bounded outputs.

The practical answer for your use case: clamp in post-processing if you need strict [0,1] bounds. The model's training loss naturally pushes predictions toward valid ranges, and the gradient benefits of the unconstrained formulation outweigh the minor post-processing cost.

v1.6.0 improved the overall training pipeline significantly, including Hungarian matcher stability fixes for non-finite entries. If you're doing further experiments with the detection head, `pip install rfdetr>=1.6.1` is worth the upgrade. Closing as explained — reopen if you find edge cases where coordinates diverge significantly in practice!
